autopayroll-CordaService/contracts/src/test/kotlin/com/autopayroll/contracts/ContractTests.kt

```diff
@@ -0,0 +1,13 @@
+package com.autopayroll.contracts
+
+import net.corda.testing.node.MockServices
+import org.junit.Test
+
+class ContractTests {
+    private val ledgerServices = MockServices()
+
+    @Test
+    fun `dummy test`() {
+
+    }
+}
```

Comment:
Delete this file if you aren't going to bother adding a test

Reply:
This class is still pointless. Why have you added a test class with no tests in it?